app/script/event/WebSocketService.js

```diff
@@ -115,6 +116,11 @@ z.event.WebSocketService = class WebSocketService {
       };
 
       this.socket.onmessage = event => {
+        if (this._pingHasExperiencedSuspiciousInactivity()) {
+          const secondsSinceLastPing = (Date.now() - this.lastPingTime) / z.util.TimeUtil.UNITS_IN_MILLIS.SECOND;
+          this.logger.warn(`Message received but ping was inactive for ${secondsSinceLastPing} seconds, reconnecting.`);
```

Comment:
Let's wrap `${secondsSinceLastPing}` inside quotes to indicate that's a variable value.

Reply:
Not sure it is relevant in that case since it's a number of seconds (and it make the line too long for prettier :D)